Mathlib/RingTheory/TwoSidedIdeal/Basic.lean

```diff
@@ -198,6 +200,35 @@ def coeAddMonoidHom : I →+ R where
   map_zero' := rfl
   map_add' _ _ := rfl
 
+/-- If `I` is a two-sided ideal of `R`, then `{op x | x ∈ I}` is a two-sided ideal in `Rᵐᵒᵖ`. -/
+@[simps]
+def op (I : TwoSidedIdeal R) : TwoSidedIdeal Rᵐᵒᵖ where
+  ringCon := I.ringCon.op
+
+@[simp]
+lemma mem_op_iff {I : TwoSidedIdeal R} {x : Rᵐᵒᵖ} : x ∈ I.op ↔ x.unop ∈ I := by
+  constructor <;> simpa [mem_iff, I.ringCon.op_iff] using I.ringCon.symm
```

Comment:
Is this `Iff.rfl`?

Reply:
Nope, thanks to https://github.com/leanprover-community/mathlib4/pull/13610/files#r2032230268